pkg/oci/utils.go

```diff
@@ -286,6 +287,35 @@ func (spec *CompatOCISpec) PodID() (string, error) {
 	return "", fmt.Errorf("Could not find pod ID")
 }
 
+func ociResourceUpdateRuntimeConfig(ocispec CompatOCISpec, config *RuntimeConfig) error {
```

Comment:
Please create instead a function:
```go
func vmConfig(ocispec CompatOCISpec, runtime RuntimeConfig) (Resources, error) {
    ...
}
```

In case there is no memory limit defined by the config.json file, we fallback to what is provided by `RuntimeConfig`. Otherwise, we take the value provided by the config.json.